packages/inspection-capture-web/src/PhotoCapture/hooks/usePhotoCaptureSightState.ts

```diff
@@ -191,10 +191,11 @@ export function usePhotoCaptureSightState({
         alreadyTakenSights: alreadyTakenSights.length,
         totalSights: captureSights.length,
         sightSelected: notCapturedSights.length > 0 ? notCapturedSights[0].label : null,
+        captureCompleted: notCapturedSights.length === 0,
```

Comment:
Shouldn't we also update the `captureCompleted` property after each sight captured and not only when fetching the inspection?

Reply:
We handle this part in this callback: 
``` js
const handleGalleryValidate = () => {
    startTasks()
      .then(() => {
        analytics.trackEvent('Capture Completed');
        analytics.setUserProperties({ captureCompleted: true });
        onComplete?.();
      })
      .catch((err) => {
        loading.onError(err);
        handleError(err);
      });
  };
  ```